test/constants.js

```diff
@@ -16,13 +16,13 @@ exports.binArgs = {
 };
 
 exports.goMessage = {
-  vite: 'running',
-  'vite-preact-compat': 'running',
-  'vite-signals': 'running',
+  vite: 'ready',
+  'vite-preact-compat': 'ready',
+  'vite-signals': 'ready',
 };
 
 exports.defaultPort = {
-  vite: 3000,
+  vite: 5173,
```

Comment:
Let's add some config to keep this on 3000

Reply:
fixed
https://github.com/preactjs/prefresh/pull/507/commits/af98c08eb09a7b846a366b359850d9254f1af51f